app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -71,11 +47,19 @@ To be eligible for a bursary, you’ll need at least one of the following:
 
 If your course combines 2 subjects (for example, English and history), at least 50% of your course content must be made up of the bursary subject to be eligible.
 
-You’ll also need to meet the bursary scheme’s individual terms and conditions.
+#### How they're paid
 
-[Find out more about your eligibility for teaching bursaries](https://www.gov.uk/government/publications/funding-initial-teacher-training-itt/funding-initial-teacher-training-itt-academic-year-2024-to-2025).
+A bursary is paid over a minimum of 10 months by your teacher training provider. They will confirm when you will get the payments. 
 
-### Scholarships
+You will not need to pay tax or National Insurance on bursary payments. 
+
+#### How to get one
+
+You do not need to apply for a bursary. When you apply for teacher training, your course provider will check if you're eligible for a bursary. If you are, your teacher training provider will confirm this in writing before you start your course. 
+
+## Scholarships
+
+Scholarships are provided by charitable organisations. They offer more than just financial benefits. For example, you can get mentoring, subject specialist events, or access to networking events. 
 
 #### Eligibility
```

Comment:
```suggestion
#### Eligibility for a scholarship
```
Currently there are two identical h4s on the page which is not good for screen reader users